cli/src/clean.rs

```diff
@@ -1,18 +1,64 @@
 use std::path::PathBuf;
 
+use anyhow::{anyhow, Context, Result};
 use clap::Parser;
+use dialoguer::Confirm;
 use log::info;
 
+use hydrophonitor_lib::{clean as clean_lib, connect};
+
+use crate::connect::connect;
+
 #[derive(Parser, Debug)]
 #[clap(about = "This command removes all deployment data from the given device's /output path")]
 pub struct Clean {
     ///Path to USB mass storage or SD card where data will be deleted from.
-    #[clap(short, long, required = true)]
-    pub device: PathBuf,
+    #[clap(short, long)]
+    device: Option<PathBuf>,
 }
 
 impl Clean {
-    pub fn clean(&self) {
-        info!("Cleaning device at {:?}", self.device);
+    pub fn clean(&mut self) -> Result<()> {
+        //create path to output folder
+        let _mount;
+        let mut output_dir: PathBuf;
+        match &self.device {
+            Some(device) => output_dir = device.clone(),
+            None => {
+                _mount = connect().with_context(|| "connecting to device failed:")?;
+                output_dir = connect::MOUNT_PATH.clone();
+            }
+        }
+        output_dir.push("output");
+
+        // Show deployments and ask for confirmation
+        match clean_lib::get_deployments_of_device(&output_dir) {
+            Some(deployments) => {
+                info!("Cleaning device at {:?}", self.device);
+
+                if !deployments.is_empty() {
+                    dbg!(deployments);
```

Comment:
Deployment directories to be cleaned could be listed at info level. Does not have to be fixed right now.